graphql-client/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/client/ReactorApollo.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.graphql.client;
+
+import static com.apollographql.apollo.ApolloCall.StatusEvent.COMPLETED;
+
+import lombok.RequiredArgsConstructor;
+import reactor.core.Disposable;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.FluxSink;
+import reactor.core.publisher.FluxSink.OverflowStrategy;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.MonoSink;
+
+import com.apollographql.apollo.ApolloCall;
+import com.apollographql.apollo.ApolloCall.StatusEvent;
+import com.apollographql.apollo.ApolloSubscriptionCall;
+import com.apollographql.apollo.api.Response;
+import com.apollographql.apollo.exception.ApolloException;
+import com.apollographql.apollo.internal.subscription.ApolloSubscriptionTerminatedException;
+import com.apollographql.apollo.internal.util.Cancelable;
+
+public final class ReactorApollo {
```

Comment:
This should live in a separate module - even better, contributed to apollographql as `apollo-reactor-support`